app.js

```diff
@@ -115,6 +116,12 @@ app.get('/waffles/custom', (request, response) => {
   response.send(getLayoutHTML(content));
 });
 
+app.get('/greet', (request, response) => {
+  let content =
```

Comment:
If you visit `/greet?name=Hello` you can reference the value of `name` in here like so:

```javascript
let name = request.query.name
```